app/models/form5655_submission.rb

```diff
@@ -10,11 +12,16 @@ def form
     @form_hash ||= JSON.parse(form_json)
   end
 
+  def user_cache_id
+    user = User.find(user_uuid)
```

Comment:
Is there a reason you can't just use `@user` here? I think you can assume the user is authenticated, right?

Reply:
No I don't suppose there is, I'll make the change.